plugins/transformerExporter.go

```diff
@@ -58,56 +61,5 @@ type exporter string
-			vat_suck.EventTransformerInitializer,
-			vow_fess.EventTransformerInitializer,
-			vow_file.EventTransformerInitializer,
-			vat_file_debt_ceiling.EventTransformerInitializer,
-			vat_file_ilk.EventTransformerInitializer,
-			vow_flog.EventTransformerInitializer,
-			yank.EventTransformerInitializer,
-		},
-		[]interface1.StorageTransformerInitializer{
-			bat_flip.StorageTransformerInitializer,
-			cat.StorageTransformerInitializer,
-			cdp_manager.StorageTransformerInitializer,
-			eth_flip_a.StorageTransformerInitializer,
-			flap_storage.StorageTransformerInitializer,
-			flop_storage.StorageTransformerInitializer,
-			jug.StorageTransformerInitializer,
-			sai_flip.StorageTransformerInitializer,
-			spot.StorageTransformerInitializer,
-			vat.StorageTransformerInitializer,
-			vow.StorageTransformerInitializer,
-		},
-		[]interface1.ContractTransformerInitializer{}
+	return []interface1.EventTransformerInitializer{yank.EventTransformerInitializer, bite.EventTransformerInitializer, cat_file_flip.EventTransformerInitializer, cat_file_vow.EventTransformerInitializer, jug_file_vow.EventTransformerInitializer, vat_fork.EventTransformerInitializer, vat_heal.EventTransformerInitializer, flop_kick.EventTransformerInitializer, vat_file_ilk.EventTransformerInitializer, vow_file.EventTransformerInitializer, flap_kick.EventTransformerInitializer, new_cdp.EventTransformerInitializer, log_value.EventTransformerInitializer, pot_file_dsr.EventTransformerInitializer, deal.EventTransformerInitializer, dent.EventTransformerInitializer, jug_drip.EventTransformerInitializer, jug_file_base.EventTransformerInitializer, vat_grab.EventTransformerInitializer, vat_move.EventTransformerInitializer, vow_fess.EventTransformerInitializer, pot_cage.EventTransformerInitializer, spot_file_mat.EventTransformerInitializer, spot_poke.EventTransformerInitializer, tick.EventTransformerInitializer, vat_frob.EventTransformerInitializer, vat_flux.EventTransformerInitializer, vat_fold.EventTransformerInitializer, cat_file_chop_lump.EventTransformerInitializer, spot_file_pip.EventTransformerInitializer, tend.EventTransformerInitializer, vat_init.EventTransformerInitializer, vat_slip.EventTransformerInitializer, flip_kick.EventTransformerInitializer, jug_file_ilk.EventTransformerInitializer, pot_file_vow.EventTransformerInitializer, vat_file_debt_ceiling.EventTransformerInitializer, vow_flog.EventTransformerInitializer, jug_init.EventTransformerInitializer, vat_suck.EventTransformerInitializer}, []interface1.StorageTransformerInitializer{cdp_manager.StorageTransformerInitializer, flop_storage.StorageTransformerInitializer, spot.StorageTransformerInitializer, jug.StorageTransformerInitializer, vow.StorageTransformerInitializer, bat_flip.StorageTransformerInitializer, eth_flip_a.StorageTransformerInitializer, vat.StorageTransformerInitializer, cat.StorageTransformerInitializer, flap_storage.StorageTransformerInitializer}, []interface1.ContractTransformerInitializer{}
```

Comment:
ugh?

Reply:
I'm kind of in favor of having each of these transformer initializers on a new line so it's easier to see what has been included, though it does make the file a lot longer. 🤷‍♀ But, don't feel super strongly either way.

Reply:
Yeah happy to add line breaks- do y'all know a way to automate that? I just copied over the generated file

Reply:
Oh right it's the generated exporter, no idea. No biggie either.